mono/metadata/assembly.c

```diff
@@ -89,7 +89,7 @@ static char **assemblies_path = NULL;
 /* Contains the list of directories that point to auxiliary GACs */
 static char **extra_gac_paths = NULL;
 
-#ifndef DISABLE_ASSEMBLY_REMAPPING
+#ifndef DISABLE_DESKTOP_LOADER
```

Comment:
Wouldn't be better instead of two negatives to have simple `#if DESKTOP_LOADER_FEATURE` or something similar?